FXSkins/src/main/resources/com/pixelduke/control/skin/fx-skins.css

```diff
@@ -47,23 +47,31 @@ TextField {
     -fx-skin: "com.pixelduke.control.skin.FXTextFieldSkin";
 }
 
-TextField {
-    -right-button-visible: true;
+.text-field {
+    -fx-padding: 0.3333333em 3em 0.3333333em 0.5833333em;
```

Comment:
why do we need to change the padding of the textfield?

Reply:
This prevents the text from going behind the right button. (The padding values are from the default jfx theme (Caspian) except the value for the right side)